cmd/server/app/server.go

```diff
@@ -344,7 +345,7 @@ func (p *Proxy) runAdminServer(o *options.ProxyRunOptions, server *server.ProxyS
 		}
 	}
 	adminServer := &http.Server{
-		Addr:           fmt.Sprintf("127.0.0.1:%d", o.AdminPort),
```

Comment:
To avoid weakening the default security posture, I would populate the new admin bind address flag to 127.0.0.1 (and document). Sorry for not noticing this before.

LGTM otherwise

Reply:
hmm, not sure if I follow. Aren't we already doing this: https://github.com/kubernetes-sigs/apiserver-network-proxy/pull/363/files/c8843948648d9e50274587a26a88bd2a797defbf#diff-803f4f0a42a022f49b840e5df162f6111f3d7b4e0790de682ef58263478f7140R330